src/app/pages/home/about/about.component.html

```diff
@@ -1 +1,41 @@
-<div>About Page</div>
\ No newline at end of file
+<div class="aboutus-overlay1">
+    <div class="aboutus-overlay1-content">
+      <div class="headline">About NTU Open Source Society</div>
```

Comment:
You can also just use the `<h4>` tag to bold this statement and to make the font-size larger. Note that we can also make use of Bootstrap's utility `mb-4` for spacing.

```suggestion
      <h2 class="mb-4">About NTU Open Source Society</h2>
```